client/logs.go

```diff
@@ -26,8 +26,10 @@ import (
 	"time"
 )
 
+// This needs to be the size of daemon.api_logs.logReaderSize, plus extra to
+// handle the timestamp, service name, and JSON syntax.
```

Comment:
Thanks for the explanatory comment.

Nit: in Go, the filename doesn't form part of the namespace, it's just the package name. So this would be referenced as just `daemon.logReaderSize`.